package.json

```diff
@@ -82,6 +83,7 @@
     "class-validator": "0.13.2",
     "cluster": "0.7.7",
     "compression": "1.7.4",
+    "cookie": "0.5.0",
```

Comment:
What do you think about using [`cookie-parser`](https://www.npmjs.com/package/cookie-parser)? We just need to add it as middleware in the server? It'll be a little cleaner.